contrib/masterfiles-stage/masterfiles-stage.sh

```diff
@@ -1,45 +1,43 @@
 #!/usr/bin/env bash
-# Authors: Ted Zlatanov <tzz@lifelogs.com>, Nick Anderson <nick@cmdln.org>, Mike Weilgart
-DIRNAME=$(dirname $0);
+# Authors: Ted Zlatanov <tzz@lifelogs.com>, Nick Anderson <nick@cmdln.org>, Mike Weilgart <mikeweilgart@gmail.com>
+mydirname="$(dirname "$0")";
 
 # Load option parsing
-source "${DIRNAME}/options.sh"
+# Sets vars: PARAMS, MASTERDIR, verbose_mode, debug_mode
+source "${mydirname}/options.sh" || { echo "options.sh couldn't be sourced" >&2 ; exit 17 ; }
 
-# Load common functionaly, upstream implementations
-source "${DIRNAME}/common.sh"
+# Load common functionality, upstream implementations
+source "${mydirname}/common.sh" || { echo "common.sh couldn't be sourced" >&2 ; exit 17 ; }
 
-MASTERDIR=$opt_deploy_dir
-PARAMS=$opt_params_file
+[ -f "$PARAMS" ] || error_exit "ERROR: Missing '$PARAMS'"
 
-# If PARAMS is unset, then we look to the default location
-#PARAMS=${PARAMS:-/opt/cfengine/dc-scripts/params.sh}
-if [ ! -f "$PARAMS" ]; then
-  error_exit "ERROR: Missing '$PARAMS'"
-fi
+source "$PARAMS"
 
-source $PARAMS
-
-  # We probably want a different temporary location for each remote repository
-  # so that we can avoid conflicts and potential confusion.
-  # Example: 
-  # ROOT="/opt/cfengine/masterfiles_staging"
-  # PARAMS="/var/cfengine/policychannel/production_1.sh"
-  # TMP=/opt/cfengine/masterfiles/staging/_tmp_var_cfengine_policychannel_production_1_sh
-
-  TRANSLATED_PARAMS=$(echo $PARAMS | tr [./] _)
-  STAGING_DIR="${ROOT}/_tmp${TRANSLATED_PARAMS}"
```

Comment:
@nickanderson, you mentioned elsewhere that putting things in `${ROOT}` broke integration with Design Center.  This is how the code was (and is) in the codebase before my merge request.  I simply moved these lines of code from here to their own function in `common.sh`.